source/common/http/conn_manager_impl.cc

```diff
@@ -1776,6 +1827,10 @@ void ConnectionManagerImpl::ActiveStream::encodeHeadersInternal(ResponseHeaderMa
     }
   }
 
+  // stream_info_.response_code_ is used by trySendLocalResponse to determine if non-informational
```

Comment:
Out of date comment? Also, for the next reader, can you add a bit more of a comment on where informational headers are sent? I assume that is in encode100Continue or something like that? It's not completely clear from a quick read why they wouldn't come through here. Along these lines, maybe rename to `non_informational_response_headers_encoded_`?